packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6361,3 +6861,214 @@ Property details
 
  * `url` can't have query or fragment parameters.
  * `index` has to be a non-negative number and cannot be duplicated in a set of ACS endpoints configured for an app.
+
+### Provisioning Connection object
+
+Defines the application provisioning connection.
+
+#### Example
```

Comment:
This doesn't seem to follow the way the other objects are written, which is of the form:
* name of the object
* description of the object
* property table
* example (without a heading)
See http://localhost:8080/docs/reference/api/apps/#application-user-credentials-object as an example

Reply:
It looks like starting from https://developer.okta.com/docs/reference/api/apps/#application-user-credentials-object and down. The format changes to have sections. I am fine with either but having separate headings allows for linking such as http://localhost:8082/docs/reference/api/apps/#application-feature-properties

Reply:
@IanHakes-okta  I looking at the example you sent and it is actually a subsection. The larger section is https://developer.okta.com/docs/reference/api/apps/#application-user-object which follows the format of 
* name of object 
* description
* Example heading, with example
* Properties Heading, with properties table
 I do use sub sections so I will update those to follow the `application-user-credentials-object`

Reply:
OK - as long as we're consistent